erpnext/public/js/controllers/transaction.js

```diff
@@ -837,6 +837,7 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 	}
 
 	validate() {
```

Comment:
added in validate to remove the disabled pricing rule

Reply:
To handle the scenario where the pricing rule is disabled after the invoice has been created